packages/manager/modules/pci/src/projects/index.js

```diff
@@ -1,21 +1,24 @@
 import angular from 'angular';
 import '@uirouter/angularjs';
 import 'oclazyload';
+import '@ovh-ux/ng-ovh-feature-flipping';
 
 const moduleName = 'ovhManagerPciProjectsLazyLoading';
 
-angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
-  /* @ngInject */ ($stateProvider) => {
-    $stateProvider.state('pci.projects.**', {
-      url: '/projects',
-      lazyLoad: ($transition$) => {
-        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+angular
+  .module(moduleName, ['ui.router', 'oc.lazyLoad', 'ngOvhFeatureFlipping'])
```

Comment:
I think the dependency should be added on the module once it's lazyloaded, here the dependency will be loaded even if it might not be needed

Reply:
i moved it into import module.
i don't notice any issues so it seems is OK.
Fixed.

Reply:
Loading of the dependency  in lazyloaded  zntry has not been removed yet though

Reply:
Fixed